src/modules/home/components/property/index.tsx

```diff
@@ -18,9 +39,9 @@ export function PropertyList() {
         py: { xs: 2, sm: 3, md: 4 },
       }}
     >
-      {data.map((item) => (
-        <ApartmentCard key={item.id} {...item} />
-      ))}
+      {propertiesData.map((item) => {
+        return <ApartmentCard key={item.id} {...item} />;
```

Comment:
Any reasons changing from `() => ()` to `() => {return ...}`?

Reply:
Initially, I was thinking of doing some if checks here but then it became unnecessary. I will turn it back.